lab-evan-erick-michael/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "lab-evan-erick-michael",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "gulp-mocha": "^3.0.1",
+    "mocha": "^3.2.0"
+  }
+}
```

Comment:
@kcirekcom @jmpaik @EWPS07 `package.json` and associated `devDependencies` look good!